pkg/pipeline/ingest/ingest_collector.go

```diff
@@ -24,6 +24,7 @@ import (
 	"fmt"
 	"github.com/netobserv/flowlogs2metrics/pkg/api"
 	"github.com/netobserv/flowlogs2metrics/pkg/config"
+	exitUtils "github.com/netobserv/flowlogs2metrics/pkg/pipeline/utils"
```

Comment:
maybe pipeline_utils or pUtils - if we add more utils it doesn't make sense to call that `exitUtils`